js/views/PageNav.js

```diff
@@ -140,7 +141,8 @@ export default class extends View {
   }
 
   navSettingsClick() {
-    // activate settings modal here
+    new SettingsModal().render().open();
```

Comment:
I think we need to keep a reference to this modal and then check if the modal is already open before launching a new one, maybe:

```
if (!this.settingsModal || !this.settingsModal.isOpen()) {
  this.settingsModal = new SettingsModal().render().open();
}
```

To be honest, since there will be at least one other piece of code that launches the Settings modal (which would be keyboard shortcuts), I think we're going to want to move the launching of the modal into some decoupled module that any interested parties can call. But... I think for the time being it's ok for pageNav to manage it.